### ISSUE TITLE
Voicing says "Maller" instead of "Smaller

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#308]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.4

**Browser**
Chrome

**Problem description**
https://github.com/phetsims/qa/issues/702

When the mass is decreased, Voicing says "As mass gets smaller, force arrows get maller"

**Steps to reproduce**
1. Enable voicing
2. Select the first 2 voicing options
3. Go to either spinner and decrease the mass. 

**Visuals**

https://user-images.githubusercontent.com/87318828/132583431-6d5ff66a-0ba0-4890-af0e-9187384729cd.mov

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.1 Safari/605.1.15
Language: en-us
Window: 1440x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @Nancy-Salpepi, it really does sound like "maller" in your recording. I just tried on Windows 10 Chrome and you can really hear the "S" in "smaller", so it is not a missing character in the string. And there are no cases of "maller" (matching word) in any of the ISLC sims. I think it is an issue with the synth.

**Comment by jessegreenberg** [comment]
This is not happening for me in macOS 11.4 Firefox 90.0.2. I am using voice "Alex" with default settings. @Nancy-Salpepi can you please comment which voice you were using? I think it sounds like "Alex" in your recording.

@terracoda @emily-phet what would you like to do about this issue? In general, what should our policy be when the synth has erroneous output? Maybe we just need to handle it on a case by case basis. This issue may be acceptable, while things like https://github.com/phetsims/gravity-force-lab-basics/issues/298 are not.

**Comment by Nancy-Salpepi** [comment]
@jessegreenberg I am using "Alex" at a rate of 1.25x in the video. I can hear the "s" of "smaller" pronounced using safari. It is with chrome that I am hearing "maller."

**Comment by jessegreenberg** [comment]
Oh thanks! Sorry, I am not sure why I misread that it was happening in Firefox. I just tried it out in macOS 11.4 with Chrome using "Alex" at 1.25x. I still was not able to produce what your synth created, I am still clearly hearing the "s" in the second "smaller".

**Comment by Nancy-Salpepi** [comment]
@jessegreenberg I think we already know that my computer has a mind of its own! Do you want me to check on my iMac?

**Comment by terracoda** [comment]
I think miss-pronunciations will be a problem sometimes.

In this case, we know there is no such thing as "maller than" so the mispronunciation should not cause a confusion in terms of learning. It's just not ideal.

I don't think we need to try and fix this one. We need to watch for things that cause completely incorrect utterances. A good example of a bad one was when we heard "Massachusetts" instead of "Mass".

**Comment by jessegreenberg** [comment]
OK thanks @terracoda, sounds good.

@Nancy-Salpepi thanks but that is OK, I think that is sufficient for this issue. Closing.
